src/misc/lv_fs.c

```diff
@@ -203,9 +203,9 @@ static lv_fs_res_t lv_fs_read_cached(lv_fs_file_t * file_p, char * buf, uint32_t
                 /*If remaining data chunk is smaller than buffer size, then read into cache buffer*/
                 res = file_p->drv->read_cb(file_p->drv, file_p->file_d, (void *)buffer, buffer_size, &bytes_read_to_buffer);
                 file_p->cache->start = file_p->cache->end + 1;
-                file_p->cache->end = file_p->cache->start + bytes_read_to_buffer - 1;
+                file_p->cache->end = (uint32_t)(file_p->cache->start + bytes_read_to_buffer - 1);
```

Comment:
Do we need to change cache->start, cache->end to `size_t`?

Reply:
I see no issue with changing them to `size_t`.

Reply:
@glueckm Do you have time to complete this modification? I believe the merger will be completed soon :)

Reply:
Sure, I can do it, but I'm not 100% sure how to proceed?
Should I change the start(end back to uint32_t and add the cast or do we keep the start/end at size_t and >I just make sure that the pull request merges with the current master?

Reply:
It is recommended to unify to size_t:
```c
typedef struct {
    size_t start;
    size_t end;
    size_t file_position;
    void * buffer;
} lv_fs_file_cache_t;
```